### ISSUE TITLE
V1.5 gives a warning with nullable strings

### ISSUE THREAD

**Issue by lookbusy1344** [dotmake-build/command-line#3]
In version 1.5 nullable string properties give a warning:

```
[CliOption(Description = "Description for Extra")]
public string? Extra { get; set; }

```

Warning	DMCLI33	DotMake.CommandLine.SourceGeneration -> The property 'Extra' has custom type 'string?' (other than supported CLR types) so it should have a public constructor or a static 'Parse' method with a string parameter, so that it can be bound/parsed as a CLI option .

I guess `string?` can be parsed automatically just like `string`

**Comment by calacayir** [comment]
Well, I rushed the release, seems missed a check with nullable but that warning should not be harmful (if you can run and the option is parsed correctly). I will fix it shortly.

**Comment by calacayir** [comment]
This is fixed now in v1.5.2
